packages/functional-tests/pages/forceAuth/index.ts

```diff
@@ -8,21 +8,21 @@ import { Credentials } from '../../lib/targets/base';
 
 export class ForceAuthPage extends BaseLayout {
   readonly path = 'force_auth';
-  context;
-  service;
+  context: any;
```

Comment:
Can this be something other than a string?

Reply:
Thanks for the call out. I think I just got lazy and assumed it was an object since it's called context.